pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md

```diff
@@ -92,30 +90,91 @@ You can define a list of CSS classes within the `<style amp-custom>` tag in the
     }
     .visible {
       opacity: 1;
-
     }
     .invisible {
       opacity: 0;
     }
     .left {
-      transform: translatex(50px)
+      transform: translatex(-50px)
     }
     .right {
-      transform: translatex(-50px)
+      transform: translatex(50px)
     }
+    button {
+      margin-top:  1rem;
+      margin-left: 1rem;
+    }
+  </style>
+</head>
+<body>
+  <amp-state id="magicBox">
+  <script type="application/json">
+    {
+      "visibleBox": {
+        "className": "visible"
+      },
+      "invisibleBox": {
+        "className": "invisible"
+      },
+      "moveLeft": {
+        "className": "left"
+      },
+      "moveRight": {
+        "className": "right"
+      }
+    }
+  </script>
+</amp-state>
```

Comment:
The indention is not correct here.
The marked block should have two spaces more indention.